guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -256,6 +256,136 @@ ifeval::["{build}" == "satellite"]
 When the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the *{project-client-name}* repository.
 endif::[]
 
+[id="creating-hosts-with-uefi-http-boot-provisioning_{context}"]
+=== Creating Hosts with UEFI HTTP Boot Provisioning
+
+You can provision hosts from {Project} using the UEFI HTTP Boot.
+This is the only method with which you can provision hosts in IPv6 network.
+
+.Procedure
+
+. On {Project}, update the `grub2-efi` package to the latest version:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {packages-update} grub2-efi
+----
+
+. Enable `foreman-proxy-http`, `foreman-proxy-httpboot`, and `foreman-proxy-tftp true` features.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario} \
+--foreman-proxy-httpboot true \
+--foreman-proxy-http true \
```

Comment:
This opens port 8000. Perhaps there should be a note somewhere that this should be open in firewalls? The HTTPS port (8443 / 9090) is typically only use by Foreman but port 8000 should be accessible by all hosts that are provisioned.

Reply:
Good point. I have added info about ports to the Planning guide. Then I added a [prerequisite](https://github.com/theforeman/foreman-documentation/pull/203/files#diff-4090459dd4762cd295ff4651b32b6440R267) to the Creating Hosts with UEFI HTTP Booting procedure to look at the section with requirements in the Planning guide.
Hopefully, in the future, we can modularize Planning and Prov guides and reuse this section with requirements to keep it closer to the creating host procedure.